### DISCUSSION TITLE
希望简化直播间新增屏蔽这个复读机功能

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4544] [功能建议 / Ideas]
![OC4L`6)$7NMLF6QNN1EA4LS](https://github.com/the1812/Bilibili-Evolved/assets/14146754/aaf9b97f-28cb-473f-9ae7-2d1757ae544b)

非常挡屏幕，很头疼，都不知道产品经理怎么想的

**Comment by GalaxySnail** [comment]
可以使用广告屏蔽插件（如 ublock origin 或 adblock 或 adblock plus），在其中添加如下规则：

```
live.bilibili.com###combo-card
```
